src/app/components/Promotions/Promotions.scss

```diff
@@ -1,10 +1,4 @@
 .promotions {
   height: 40vw;
-  position: relative;
-  .spin-position {
-    position: absolute;
-    top: 50%;
-    right: 50%;
-    left: 50%;
-  }
+  // position: relative;
 }
```

Comment:
Looks like we can move this line into .js file and make varible for this style rule